Assignment2/Extract_Bits/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+/// <summary>
+/// Gets the number, start and stop bits then extracts the bits and provides the decimal value of it.
+/// </summary>
+class BitsExtraction{
+
+    /// <summary>
+    /// This Main method gets the inputs from the user.
+    /// </summary>
+    static void Main(){
+        Console.WriteLine("Enter the input number:");
+        var userInput = int.Parse(Console.ReadLine());
+        string userInputBinary = Convert.ToString(userInput,2);
+        //Gets a valid starting bit
+        int startBit;
+        do{
```

Comment:
Documentation missing. Add an overview of what the loop is doing. It will be difficult to understand if there is no documentation.

Reply:
Documentation added for all loops.